File: Presentations/WG2_11/GlassBR_Code/fromGOOL/python/InputFormat.py
from __future__ import print_function
import sys
import math
import InputParameters


def get_input(filename, inparams):
    infile = open(filename, "r")
    inparams.a = float(infile.readline());
    inparams.b = float(infile.readline());
    inparams.t = float(infile.readline());
    inparams.gt = int(infile.readline());
    inparams.w = float(infile.readline());
    inparams.tnt = float(infile.readline());
    inparams.sdx = float(infile.readline());
    inparams.sdy = float(infile.readline());
    inparams.sdz = float(infile.readline());
    inparams.pbtol = float(infile.readline());
    infile.close()

File: Presentations/WG2_11/GlassBR_Code/fromGOOL/cpp/InputFormat.cpp
#include "InputFormat.hpp"

#include "InputParameters.hpp"

#include <algorithm>
#include <iostream>
#include <fstream>
#include <iterator>
#include <string>
#include <math.h>
#include <sstream>
#include <vector>

using namespace GlassBR;
using std::string;
using std::vector;
using std::ifstream;
using std::ofstream;

void GlassBR::get_input(string filename, InputParameters &inparams) {
    ifstream infile;
    infile.open(filename, std::fstream::in);
    infile >> inparams.a;
    infile >> inparams.b;
    infile >> inparams.t;
    infile >> inparams.gt;
    infile >> inparams.w;
    infile >> inparams.tnt;
    infile >> inparams.sdx;
    infile >> inparams.sdy;
    infile >> inparams.sdz;
    infile >> inparams.pbtol;
    infile.close();
}

